server/src/main/java/io/druid/curator/CuratorConfig.java

```diff
@@ -66,4 +69,14 @@ public void setEnableCompression(Boolean enableCompression)
   {
     this.enableCompression = enableCompression;
   }
+  
+  public Boolean getEnableAcl()
+  {
+    return enableAcl;
+  }
+
+  public void setEnableAcl(Boolean enableAcl)
```

Comment:
same question about `boolean` (or else this will throw NPE on being passed NULL, right?)